README.md

```diff
@@ -1,18 +1,9 @@
 # ProjectTides
 
 To start your Phoenix server:
+  * `docker-compose config` Check if docker config and env variables are set properly. 
+  * `docker-compose up -d` Start the container.
+  * `mix setup` Setup the project.
+  * `mix phx.server` Start Phoenix endpoint 
 
-  * Setup the project with `mix setup`
-  * Start Phoenix endpoint with `mix phx.server`
-
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
-
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
-
-## Learn more
-
-  * Official website: https://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Forum: https://elixirforum.com/c/phoenix-forum
-  * Source: https://github.com/phoenixframework/phoenix
+Now you can visit [`localhost`](http://localhost) from your browser.
```

Comment:
In the `docker-compose` file, the port specified is 4000. Either we update `docker-compose` to use port 80 or add instructions that while using docker access via http://localhost:4000.

Reply:
Agree 👍, going with the default `http://localhost:4000`.